### PR TITLE
Update deploy.yaml

### PR DESCRIPTION
**Pull request by ShovelHand**
I don't see the parameters to add urls to namespaces when I look at a pod in OpenShift, so I added the parameters to deploy.yaml. I'm actually kind of guess that this was what was missing, so here it is for your review.

### CODE REVIEW

**Comment by w8896699** [comment]
No, there's nothing todo with this deployment.yaml.  Change need to be made in https://github.com/bcgov-c/tenant-gitops-platform-registry/blob/c0762900a2dba9a259cf47dceef79baf2e4e97de/platform-registry-app/base/deployment.registry-api.yaml#L113.

**Comment by w8896699** [comment]
And as I mentioned in RC, we only need one URL for each cluster(currently you have 4)

**Review by w8896699 — CHANGES_REQUESTED** [review]
Please look at above two comments

**Comment by ShovelHand** [comment]
So there's no doubt; you're idea is that we have two base  URLs, one for some Gold  namespace, and One for a silver namespace, then we just programatically add "-prod", or "-dev" or whatever to it as is appropriate, right @w8896699?

**Comment by w8896699** [comment]
Yes

**Review by w8896699 — CHANGES_REQUESTED** [review]
I suggest that you revert the previous commit and redo the whole task in a new pr as the whole logic now has been changed. https://github.com/bcgov/platform-services-registry/pull/672

File: api/src/controllers/quota.ts [hunk 1]
```diff
@@ -77,24 +77,15 @@ export const getAllAllowedQuotaSize = async (req: Request, res: Response) => {
...
-    const prodGoldUrl = process.env.PRODUCTION_NAMESPACE_GOLD_URL || "";
-    const devGoldUrl = process.env.DEVELOPMENT_NAMESPACE_GOLD_URL || "";
-    const testGoldUrl = process.env.TEST_NAMESPACE_GOLD_URL || "";
-    const toolsGoldUrl = process.env.TOOLS_NAMESPACE_GOLD_URL || "";
-    // silver
-    const prodSilverUrl = process.env.PRODUCTION_NAMESPACE_SILVER_URL || "";
-    const devSilverUrl = process.env.DEVELOPMENT_NAMESPACE_SILVER_URL || "";
-    const testSilverUrl = process.env.TEST_NAMESPACE_SILVER_URL || "";
-    const toolsSilverUrl = process.env.TOOLS_NAMESPACE_SILVER_URL || "";
+    const goldUrl = process.env.GOLD_CLUSTER_URL || "";
```
[the remark this answers is not in the document]
Comment (ShovelHand) [reply api/src/controllers/quota.ts:85]: Yes

**Comment by ShovelHand** [comment]
I am not redoing this task as a new PR. That is a needless reproduction of effort on something that has already gone on for way too long.

File: web/src/components/profileEdit/QuotaCard.tsx [hunk 2]
```diff
@@ -26,7 +26,6 @@ import {
   ProjectNamespaceResourceQuotaSize,
 } from '../../types';
 import { BaseIcon } from '../common/UI/Icon';
-import useRegistryApi from '../../hooks/useRegistryApi';
```
Comment (w8896699) [inline web/src/components/profileEdit/QuotaCard.tsx:29]: clean up function in useRegistryApi as well

Comment (w8896699) [reply web/src/components/profileEdit/QuotaCard.tsx:29]: and in web/src/views/ProfileEdit.tsx

File: api/src/router/routes/quota.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ const router = express.Router();
 // Fetches raw quota specifications from DB
 // This endpoint is not being utilized by the front end; implemented for future use
 router.get("/", asyncMiddleware(fetchQuota));
-router.get("/namespaceUrls", asyncMiddleware(fetchNamespaceUrls));
+router.get("/namespaceUrls", asyncMiddleware(getClusterUrls));
```
Comment (w8896699) [inline api/src/router/routes/quota.ts:34]: Are we still need changes in api?

**Review by w8896699 — CHANGES_REQUESTED** [review]
Almost there

**Comment by ShovelHand** [comment]
I took a good, close look, and I think we're all good now. I thought I'd get you to have a last look before merging @w8896699

**Review by w8896699 — APPROVED** [review]

**Pull request merged**